lcls-twincat-pmps/PMPS/MajorComponents/PhotonEnergyInterface/FB_SXU.TcPOU

```diff
@@ -245,7 +245,7 @@ END_VAR
     </Property>
     <Action Name="UndAdrUpdate" Id="{9de2fd9a-9f44-4b71-9e83-edd38b0d7037}">
       <Implementation>
-        <ST><![CDATA[fbSegment[21] := 0;
+        <ST><![CDATA[fbSegment[21] := ADR(fbSegment_21);
```

Comment:
This looks correct to me
I double-checked the rest of the file to make sure 21 is included in the calculations, and it is (when adr is nonzero)